src/BenchmarkDotNet/Toolchains/MonoWasm/WasmToolChain.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using JetBrains.Annotations;
+using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Loggers;
+
+namespace BenchmarkDotNet.Toolchains.MonoWasm
+{
+    [PublicAPI]
+    public class WasmToolChain : Toolchain
+    {
+        [PublicAPI] public static readonly IToolchain NetCoreApp50Wasm = From(NetCoreAppSettings.NetCoreApp50, new WasmSettings(null, null, null));
```

Comment:
will it work if we don't specify all the paths? If not, this property should be removed

```suggestion
        [PublicAPI] public static readonly IToolchain NetCoreApp50Wasm = From(NetCoreAppSettings.NetCoreApp50, new WasmSettings(null, null, null));
```